src/containers/services/ServicesSection.js

```diff
@@ -22,20 +22,31 @@ import {
 
 const Content = styled.div`
   display: block;
-  margin-bottom: 350px;
-  margin-top: 150px;
-  max-width: 500px;
+  margin: 80px 0 100px 0;
+  max-width: 100%;
+
+  @media only screen and (min-width: 768px) {
+    max-width: 50%;
+  }
+
+  @media only screen and (min-width: 1020px) {
+    margin: 150px 0 350px 0;
+  }
 `;
 
 const ArtSectionOuterWrapper = styled.div`
-  display: flex;
-  flex: 0 0 auto;
-  flex-direction: row;
-  height: 375px;
-  justify-content: center;
-  margin-top: -375px;
-  min-width: ${PAGE_SECTION_MIN_WIDTH}px;
-  position: relative;
+  display: none;
```

Comment:
I'd rather keep this visible until we have responsive designs to work off of. Can you try making it work?

Reply:
Try getting it to look roughly like this, but better:

![screen shot 2018-05-14 at 12 53 27 pm](https://user-images.githubusercontent.com/440299/40020043-dd336ce4-5775-11e8-86e2-79c74ac07fa5.png)